documentation/course/e2/README.md

```diff
@@ -1032,7 +1103,7 @@ Słownik pojęć dla jest w [dokumencie z wymaganiami](../e1/README.md#słownik-
 
 ### Konto
 
-Względem modelu ze specyfikacji wymagań, dodano pole `lastModified`, które przechowuje datę ostatniej modyfikacji konta. Będzie ono wykorzystywane do realizacji wymagań dotyczących bezpieczeństwa, tj. aby zapewnić, że po zmianie danych logowania, bądź jawnym żądaniu zakończenia wszystkich sesji użytkownika, odświeżenie tokenu dostępowego będzie wymagało ponownego zalogowania.
+Względem modelu ze specyfikacji wymagań, dodano pole `lastModified`, które przechowuje czas ostatniej modyfikacji konta. Będzie ono wykorzystywane do realizacji wymagań dotyczących bezpieczeństwa, tj. aby zapewnić, że po zmianie danych logowania, bądź jawnym żądaniu zakończenia wszystkich sesji użytkownika, odświeżenie tokenu dostępowego będzie wymagało ponownego zalogowania. Ponadto zmieniono `id` na typ tekstowy, aby zastosować UUID, który w przeciwieństwie do liczby całkowitej nie ujawnia informacji o liczbie kont w systemie. Wszystkie typy zamieniono też na ich odpowiedniki z wybranego do implementacji języka programowania.
```

Comment:
> Ponadto zmieniono `id` na typ tekstowy, aby zastosować UUID, który w przeciwieństwie do liczby całkowitej nie ujawnia informacji o liczbie kont w systemie. Wszystkie typy zamieniono też na ich odpowiedniki z wybranego do implementacji języka programowania.

We discussed this today, thats also a general thing that should be put elsewhere.

Reply:
Moved above, good idea 👍